.github/workflows/commit-msg-chk.yaml

```diff
@@ -11,4 +11,4 @@ jobs:
     steps:
       - name: check-for-cc
         id: check-for-cc
-        uses: agenthunt/conventional-commit-checker-action@v1.0.0
+        uses: agenthunt/conventional-commit-checker-action@v2.0.0
```

Comment:
Note: We could get dependabot to automate these updates - have we considered using dependabot previously?

Reply:
Interesting. You mean for all PRs? Can you share a link to the doc which describes this?

Reply:
I've added a new commit with a dependabot config

Reply:
Ah, I see. I misunderstood your first comment. I thought that you said that we could use Dependabot to replace the conventional-commit checker somehow.

Thanks!
+ 1 from me to manage updates for dependencies automatically